pennylane/vqe/vqe.py

```diff
@@ -129,6 +160,119 @@ def __str__(self):
 
         return "\n+ ".join(terms)
 
+    def obs_data(self):
```

Comment:
Would also suggest making this a private method, or perhaps even a private property?

Reply:
Yes, this would make much more sense! I think a private method is best. However, Pylint doesn't like it when we call a private method outside of the class in which it is defined. We have to call this method over in `operations.py`, which makes Pylint angry. I'm not totally sure how to resolve this 🤔